src/Components/CollectionsHubRails/ArtistSeriesRail/ArtistSeriesEntity.tsx

```diff
@@ -23,11 +23,11 @@ export const ArtistSeriesEntity: React.FC<ArtistSeriesEntityProps> = ({
   } = member
   const bgImages = hits.map(hit => hit.image.url)
   const imageSize =
-    bgImages.length === 1 ? 265 : bgImages.length === 2 ? 131 : 85
+    bgImages.length === 1 ? 221 : bgImages.length === 2 ? 109 : 72
   return (
-    <Container p={0.5} m={0.5}>
+    <Container px={2} pt={2} pb={1.6} m={1}>
```

Comment:
the only values that we should set inside of these spacing props are values [from our theme](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L116-L138)